docs/design/0002-har-analysis-toolkit-plan-annotated.md

```
@@ -0,0 +1,228 @@
+## Technical solution
+
+
+
+<!-- -->
+
+> **✎ Notes** · `SPEC › Technical solution`
+> _(your notes here — replace this line)_
+
+### Architecture overview
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)